tests/test_strategy_infinite_bus.py

```diff
@@ -333,3 +333,66 @@ def testing_event_market_cycle_posting_bids(bus_test4, area_test4):
     assert bus_test4._bids[area_test4.test_market.id][-1].energy == sys.maxsize
     assert bus_test4._bids[area_test4.test_market.id][-1].price == 25 * sys.maxsize
     ConstSettings.IAASettings.MARKET_TYPE = 1
+
+
+"""TEST5"""
+
+
+@pytest.fixture()
+def area_test5():
+    return FakeArea(0)
+
+
+@pytest.fixture()
+def bus_test5(area_test5):
+    c = InfiniteBusStrategy(energy_rate_profile="src/d3a/resources/SAM_SF_Summer.csv")
+    c.area = area_test5
+    c.owner = area_test5
+    return c
+
+
+def test_global_market_maker_rate_profile(bus_test5, area_test5):
+    assert isinstance(GlobalConfig.market_maker_rate, dict)
+
+
+"""TEST6"""
+
+
+@pytest.fixture()
+def area_test6():
+    return FakeArea(0)
+
+
+@pytest.fixture()
+def bus_test6(area_test6):
+    c = InfiniteBusStrategy(energy_rate_profile="src/d3a/resources/SAM_SF_Summer.csv")
+    c.area = area_test6
+    c.owner = area_test6
+    return c
+
+
+def test_infinite_bus_selling_rate_set_as_profile(bus_test6, area_test6):
+    bus_test6.event_activate()
+    assert isinstance(bus_test6.energy_rate, dict)
+
+
+"""TEST7"""
+
+
+@pytest.fixture()
+def area_test7():
+    return FakeArea(0)
```

Comment:
`area_test5/6/7` are all the same. Why do we need to create separate fixtures for them?

Reply:
I followed the same pattern as before, may be thought to have isolation of area object for each unittest, also I tested no now with common `area_test` fixtures and nothing complained, so will adapt your suggestion.

Reply:
If I'm not wrong, the fixtures created by pytest have a default `scope="function"`, so they should be reset before each test function runs.